pysat/utils/io.py

```diff
@@ -396,7 +862,7 @@ def load_netcdf_pandas(fnames, strict_meta=False, file_format='NETCDF4',
         file_format keyword passed to netCDF4 routine.  Expects one of
         'NETCDF3_CLASSIC', 'NETCDF3_64BIT', 'NETCDF4_CLASSIC', or 'NETCDF4'.
         (default='NETCDF4')
-    epoch_name : str
+    epoch_name : str or NoneType
```

Comment:
Should the default be updated to None to trigger the warning below. Warning is triggered by default when calling from `load_netcdf`

Reply:
No strong feelings.

Reply:
I've confirmed that `load_netcdf` default is still None. Also, `load_netcdf_pandas` didn't exist in previous versions. Which means no history of defaults to worry about at this level.